packages/links/links.0.9.7/opam

```diff
@@ -15,7 +15,7 @@ build: [
 ]
 
 depends: [
-  "ocaml" {>= "4.08.0"}
+  "ocaml" {>= "4.08.0" & < "5.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.08.0" & < "5.0~"}
```

Just a minor tweak - the tilda will be the lowest version of 5.0, which will catch any beta releases etc.

Reply:
Thanks!